src/utils.py

```diff
@@ -81,6 +81,15 @@ def get_file_mtime(string):
 def is_file_dir(string):
     return os.path.isdir(string)
 
+@cached
```

Comment:
I don't think there's much benefit from caching this method, the cache it's only effective during the duration of the page loading operation and later discarded (we wouldn't want to keep icons for the whole FS in memory).

We should drop it the cache for this function, or... re-organize this somehow so that we can at least cache each icon for each type, and (maybe save some system memory)...